client/app/visualizations/counter/counter-editor.html

```diff
@@ -8,6 +8,12 @@
     </li>
   </ul>
   <div ng-show="currentTab == 'general'" class="m-t-10 m-b-10">
+      <div class="form-group">
+          <label class="col-lg-6">Counter Label</label>
+          <div class="col-lg-6">
+            <input type="text" ng-model="visualization.options.counterLabel" class="form-control" ng-trim="false">
```

Comment:
Could you add a placeholder attribute that shows that the counter label defaults to the visualization name?

Reply:
@jezdez Added :)